source/testing/testing-components.md

```diff
@@ -258,7 +258,7 @@ is being returned from the service is reflected in the component output.
 ```tests/integration/components/location-indicator-test.js
 test('should reveal current location', function(assert) {
   this.render(hbs`{{location-indicator}}`);
-  assert.equal(this.$().text().trim(), 'You currently are located in New York, USA');
+  assert.equal(this.$().text().trim(), 'Location: New York, USA');
```

Comment:
could you split it into a different PR? if it's too much hassle I'll merge as is